libs/k8s-client/src/ApiClient/PodsApiClient.php

```diff
@@ -31,22 +32,32 @@ public function getStatus(string $name, array $queries = []): Pod
         return $this->apiClient->request($this->baseApi, 'readStatus', Pod::class, $name, $queries);
     }
 
-    public function readLog(string $name, array $queries = []): string
+    public function readLog(string $name, array $queries = []): StreamInterface
     {
-        // do not use $this->request() wrapper, method does not return a class
-        $result = $this->baseApi->readLog($this->apiClient->getK8sNamespace(), $name, $queries);
+        $client = StreamClient::getInstance();
```

Comment:
Tohle bohuzel nebude fungovat, kvuli tem podelanejm singletonum, co to pouziva, protoze obcas potrebujem mit vic nez jednoho klienta a kazdy s jinou konfiguraci. Je potreba si toho klienta normalne injectnout pres constructor, aby se pouziva vzdy ta stejna/spravna instance, ne ta naposledy nakonfigurovana.